packages/karma-esm/src/setup-es-dev-server.js

```diff
@@ -7,14 +8,36 @@ const regexpKarmaLoaded = /window\.__karma__\.loaded\(\);/gm;
 const regexpScriptSrcGlobal = /<script type="module"[^>]*src="([^"]*)"/gm;
 const regexpScriptSrc = /<script type="module"[^>]*src="([^"]*)"/m;
 
+/**
+ * Load importMap content from a filepath
+ * @param {string} path
+ * @returns {Promise<string>}
+ */
+async function loadImportMap(path) {
+  return new Promise((resolve, reject) => {
```

Comment:
Using util.promisify for this is more idiomatic

Reply:
Done